lib/active_record/connection_adapters/cockroachdb_adapter.rb

```diff
@@ -500,8 +503,18 @@ def column_definitions(table_name)
             dtype = field[1]
             field[1] = crdb_fields[field[0]][2].downcase if re.match(dtype)
             field[7] = crdb_fields[field[0]][1]&.gsub!(/^\'|\'?$/, '')
+            field[9] = true if crdb_fields[field[0]][3]
```

Comment:
this is so hacky, lol!

Reply:
yeah idk, kinda surprising that there's no way to access the fields by name!

Reply:
just the whole `NULL as is_hidden` then overwriting it later is *chefs kiss*